activerecord/lib/active_record/autosave_association.rb

```diff
@@ -91,8 +91,10 @@ module ActiveRecord
   #   post.save # => saves both post and comment
   #
   #   post = Post.create(title: 'ruby rocks')
-  #   post.comments.create(body: 'hello world')
-  #   post.save # => saves both post and comment
+  #   comment = post.comments.create(body: 'hello world')
+  #   post.title += "!"
+  #   comment.body += "!"
+  #   post.save # => saves only post and ignores comment
```

Comment:
"ignores" is a bit negative, like we're criticising the behaviour.

```suggestion
  #   post.save # => saves post, but not comment
```